tools/go-cli/README.md

```diff
@@ -1,15 +1,9 @@
-To build the wsk CLI run ant buildGoCLI from this directory.
+To build the wsk CLI run the following command from the OpenWhisk home directory
 
-$ cd tools/go-cli
-$ ant buildGoCLI
+$ gradle :tools:go-cli:distDocker
 
-The build will produce the cli in ../bin/go-cli/$platform/$architecture/wsk
-
-The $platform is linux, mac, or windows.
-
-The $architecture is 386, amd64, arm, arm64, etc.
+The build will produce the cli in ../bin/go-cli/wsk
 
 To get the help run wsk --help
 
 The WSK_CLI_DEBUG can to be used set to "true" to provide debug information.
```

Comment:
@dubeejw - Not sure if this line is needed, and it is somewhat confusing.  This env variable is not related to building the Go CLI, but when running it.  Also, the -d option is preferred as it is simpler to enable/disable on a per command basis.

The Go CLI produces multiple binaries, but only copies one.  Probably should note that somehow.

I think the build command can only be run from specific terminal contexts.  That is, running that command from a Windows terminal won't work.  Need to mention here?